code/datums/mind.dm

```diff
@@ -719,6 +723,23 @@
 					log_admin("[key_name(usr)] tried and failed to give [current] an uplink.")
 				else
 					log_admin("[key_name(usr)] gave [current] an uplink.")
+			if("toggleexploitables")
+				if (has_exploitables_override)
```

Comment:
this and 3 lines below can be consolidated into `has_exploitables_override = !has_exploitables_override`